### DISCUSSION TITLE
Reset "always" option for unknown fingerprint

### DISCUSSION THREAD

**Discussion by jpbranski** [iterate-ch/cyberduck#15167] [Q&A]
_This is not the bug people were having about the "always" option not working._

I was just wondering what the method was to reenable that prompt. When searching for it, all searches just would lead to the issue regarding the bug.

Thanks!

**Comment by dkocher** [comment]
You have to remove the known host entry from `~ /.ssh/known_hosts`.

**Reply by jpbranski** [reply]
I actually did try that but it didn't cause it to show back up again. Appreciate it though!
